plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -40,8 +41,9 @@ var _ = admission.Interface(&persistentVolumeLabel{})
 type persistentVolumeLabel struct {
 	*admission.Handler
 
-	mutex      sync.Mutex
-	ebsVolumes aws.Volumes
+	mutex            sync.Mutex
+	ebsVolumes       aws.Volumes
+	gceCloudProvider *gce.GCECloud
```

Comment:
Can we avoid putting provider-specific fields in this provider-independent library?